pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -347,8 +347,9 @@ func TestEventListenerValidate_error(t *testing.T) {
 					Bindings: []*v1alpha1.EventListenerBinding{{Name: "tb", Kind: v1alpha1.NamespacedTriggerBindingKind, Ref: "tb"}},
 					Template: v1alpha1.EventListenerTemplate{Name: "tt"},
 					Interceptors: []*v1alpha1.EventInterceptor{{
-						GitHub: &v1alpha1.GitHubInterceptor{},
-						GitLab: &v1alpha1.GitLabInterceptor{},
+						GitHub:    &v1alpha1.GitHubInterceptor{},
+						GitLab:    &v1alpha1.GitLabInterceptor{},
+						Bitbucket: &v1alpha1.BitBucketInterceptor{},
```

Comment:
I don't think adding it here is doing anything since this is just testing the validation fails with multiple interceptors

Reply:
@dibyom I forgot to mention that adding `BitBucket` here will cover this [scenario](https://github.com/tektoncd/triggers/pull/578/files#diff-559661e0e83bb92fa19d37cc32078347R104-R106) 
The point of adding here is to cover the unit test so shall we add it back 

Let me know on this

Reply:
hmm, seems ok for now but  wondering if we should have a separate test for that...